lib/chat_api/messages.ex

```diff
@@ -127,4 +129,13 @@ defmodule ChatApi.Messages do
         dynamic
     end)
   end
+
+  defp after_message_created({:ok, message} = params) do
+    Workers.MessageCreatedActions.new(%{"id" => message.id})
```

Comment:
verrrry minor stylistic preference, but when i'm using pipes i prefer to start with the initial argument at the top like so:
```suggestion
    %{"id" => message.id}
    |> Workers.MessageCreatedActions.new()
```